client/angular-marketplace/.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
@jeannsensei I have a doubt here, whta happen if this rules doesn't match the Visual Studio formating? I think it woul be better if we create a fault tolerance set of rules at the beggining, or share the VS configuration file that match this cases (e.g. https://github.com/nitin27may/mean-docker/blob/master/.vscode/settings.json)

Reply:
Good suggestion. Let me check that and I'll update.

Reply:
Let's tackle this in another PR.